content/how-to/remediate-alerts/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Remediate alerts"
-description: "How to remediate alerts"
```

Comment:
Actually, this was a good lead and description. They're different from the title and they accurately describe the page. I wouldn't remove these.